src/minisat/mtl/Vec.h

```diff
@@ -42,9 +41,9 @@ class vec {
     int sz;
     int cap;
 
-    // Don't allow copying (error prone):
-    vec<T>&  operator = (vec<T>& other) = delete;
-             vec        (vec<T>& other) = delete;
+    vec<T>&  operator = (const vec<T>& other) {
+                static_assert(std::is_trivially_copyable<T>::value, "mtl::vec elements need to be trivially copyable");
```

Comment:
It looks like some compilers (clang) can compile the template code if the copy assignment is never called from the code.  I thought this is a feature, so I left it to the place where it is actually required.  But I didn't do a lot of research on this.  At the moment the code does not require this (as far as I know), since g++-9 was complaining about the copy assignment even in cases where it was never called.